pkg/stis/calstis/cs1/doblev.c

```diff
@@ -223,7 +229,7 @@ static FILE *BlevOpen (char *fname, int extver, int *status) {
 
 	    } else {
 
-		fclose (fp);
+	        (void)fcloseWithStatus(&fp);
 		printf ("ERROR    File %s already exists.\n", fname);
 		*status = OPEN_FAILED;
 		fp = NULL;
```

Comment:
fp has already been set to NULL in fcloseWithStatus

Reply:
I did notice this, I forgot to delete it. Doesn't matter too much.